tests/bootstrap.php

```diff
@@ -29,9 +30,13 @@
     require $root . '/config/bootstrap.php';
 }
 
+// Ensure default test connection is defined
+if (!getenv('db_dsn')) {
+    putenv('db_dsn=sqlite:///:memory:');
+}
 Configure::write('App', [
+    'namespace' => 'CsvView\Test\App',
     'paths' => [
-        'templates' => [dirname(__FILE__) . DS . 'test_app' . DS . 'TestApp' . DS . 'Template' . DS],
+        'templates' => [dirname(__FILE__) . DS . 'test_app' . DS . 'templates' . DS],
     ]
-]);
-Plugin::load('CsvView', ['path' => dirname(dirname(__FILE__)) . DS]);
+ ]);
```

Comment:
Indentation seems off here.

Reply:
maybe it was. if you see i removed code, not added